src/test/java/mate/academy/springbootapp/service/impl/CsvFileParserServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package mate.academy.springbootapp.service.impl;
+
+import java.io.FileNotFoundException;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.List;
+import mate.academy.springbootapp.dto.ReviewDto;
+import mate.academy.springbootapp.mapper.ReviewMapper;
+import mate.academy.springbootapp.service.FileParserService;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CsvFileParserServiceImplTest {
+    private static final String FILEPATH = "src/test/resources/reviews-test.csv";
+    private FileParserService csvFileParserService;
+
+    @Before
+    public void setUp() {
+        csvFileParserService = new CsvFileParserServiceImpl(new ReviewMapper());
+    }
+
+    @Test
+    public void parseCsvFileOk() {
+        var reviewDto1 = new ReviewDto();
+        reviewDto1.setId(1L);
+        reviewDto1.setProductId("B001E4KFG0");
+        reviewDto1.setUserId("A3SGXH7AUHU8GW");
+        reviewDto1.setProfileName("delmartian");
+        reviewDto1.setHelpfulnessNumerator(1);
+        reviewDto1.setHelpfulnessDenominator(1);
+        reviewDto1.setScore(5);
+        var time1 = Instant.ofEpochMilli(1303862400)
+                .atZone(ZoneId.systemDefault()).toLocalDateTime();
+        reviewDto1.setTime(time1);
+        reviewDto1.setSummary("Good Quality Dog Food");
+        reviewDto1.setText("I have bought several of the Vitality canned dog food "
+                + "products and have found them all to be of good quality. The product "
+                + "looks more like a stew than a processed meat and it smells better. "
+                + "My Labrador is finicky and she appreciates this product better than  most.");
```

Comment:
we don't need this whole text for testing. Let's cut this field to one small sentence

Reply:
done